xrayvision/tests/test_rhessi_visibility.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import pytest
+from astropy.convolution import Gaussian2DKernel
+
+from ..Visibillity import RHESSIVisibility
+
+
+class TestRHESSIVisibility(object):
+
+    # Typical map sizes even, odd
+    @pytest.mark.parametrize("N,M", [(65, 65), [64, 64]])
+    def test_from_map(self, N, M):
+        # Set up empty map
+        empty_map = np.zeros((N, M))
+
+        # Calculate full u, v coverage so will be equivalent to a discrete Fourier transform (DFT)
+        u, v = np.meshgrid(np.arange(M), np.arange(M))
+        uv_in = np.array([u, v]).reshape(2, N*M)
+        vis_in = np.zeros(N*M, dtype=complex)
+
+        vis = RHESSIVisibility(uv_in, vis_in)
```

Comment:
There's no point in testing this as it's already tested in the base class, what you want to test here wold be the creation of the class with the additional information.

Reply:
My intention was there to detect if in the future someone accidentally messes up the inherited parts in the RHESSIVisibility class, but if you think is unnecessary, I remove it.